demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -850,19 +926,23 @@ def test_upload_custom_packs_from_config_file(self, mocker):
         # prepare
         mock_api_client(mocker)
         mocker.patch.object(API_CLIENT, 'upload_content_packs')
+        mocker.patch.object(Pack, 'is_server_version_ge', return_value=False)
         mocker.patch.object(tools, 'update_server_configuration', return_value=(None, None, {}))
         mocker.patch.object(Uploader, 'notify_user_should_override_packs', return_value=True)
 
         # run
         status_code = click.Context(command=upload).invoke(
             upload, input_config_file=f'{git_path()}/demisto_sdk/commands/upload/tests/data/xsoar_config.json')
 
+        # validate
+        disable_verification_call_args = tools.update_server_configuration.call_args_list[0][1]
+        enable_verification_call_args = tools.update_server_configuration.call_args_list[1][1]
+
+        assert disable_verification_call_args['server_configuration'][constants.PACK_VERIFY_KEY] == 'false'
+        assert constants.PACK_VERIFY_KEY in enable_verification_call_args['config_keys_to_delete']
         assert status_code == 0
 
         uploaded_file_path = API_CLIENT.upload_content_packs.call_args[1]['file']
-        skip_verify = API_CLIENT.upload_content_packs.call_args[1]['skip_verify']
-
-        assert skip_verify == 'true'
         assert 'uploadable_packs.zip' in str(uploaded_file_path)
 
     @pytest.mark.parametrize(argnames='input', argvalues=[TEST_PACK_ZIP, CONTENT_PACKS_ZIP])
```

Comment:
Please update the doc string (the Then section)

Reply:
done